### ISSUE TITLE
Tooltip icon inaccessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by cjkarle** [carbon-design-system/carbon#7382]
## Environment

> Operating system

Win 10 latest

> Browser

Edge 87.0, Firefox 83.0

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
Tooltip circle+i icon is insufficiently visible to HCM users:

<img width="256" alt="Screen Shot 2020-12-01 at 5 26 52 PM" src="https://user-images.githubusercontent.com/75339318/100804461-ea428280-33fa-11eb-90c4-c1a529379213.png">

> What version of the Carbon Design System are you using?

7.24.0

**Comment by tw15egan** [comment]
I see you're running `7.24` of the react components, which version of `carbon-components` are you running? Should have been fixed in https://github.com/carbon-design-system/carbon/pull/7031 and included in [v10.23.0](https://github.com/carbon-design-system/carbon/releases/tag/v10.23.0)

**Comment by cjkarle** [comment]
We're on 10.24
```
    "carbon-components": "^10.24.0",
    "carbon-components-react": "^7.24.0",
    "carbon-icons": "^7.0.7",
```

**Comment by cjkarle** [comment]
Sorry, rechecked and the poor contrast icons are Firefox only, Edge looks good -- updated the original description

**Comment by tw15egan** [comment]
@cjkarle can you pull down `10.25`/`7.25` and see if the issue still exists? I think the latest release had some more fixes.

Also, on firefox, make sure you're on v80+ and have enabled `prefers-contrast` enabled (type `about:config` into the address bar and search for `prefers-contrast`)

**Comment by stanislavgeorgiev** [comment]
Same here. Edge is working fine but Firefox has the icons and toggles disappearing 

Firefox:
![image](https://user-images.githubusercontent.com/1253469/102128224-8f0d8880-3e1b-11eb-9c77-ffe740988804.png)

Edge:
![image](https://user-images.githubusercontent.com/1253469/102128459-e90e4e00-3e1b-11eb-8e0b-830b54690d30.png)

I have been experimenting with https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors and it looks like they do work

**Comment by tw15egan** [comment]
Just checked on my Windows 10 machine and this issue is resolved in both Firefox (with `prefers-contrast` enabled) and Edge
